Jenkinsfile.nightly

```diff
@@ -83,6 +83,7 @@ pipeline {
                     dir("${WORKING_DIR}") {
                         archiveArtifacts artifacts: 'test/logs/**/*.log', allowEmptyArchive: true
```

Comment:
@radtriste I didn't archive the history log file in the end as the sh command will remove it and I don't think it will be very necessary as you already said. What do you think?

Reply:
fair enough